src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, DrawAble {
+    private String name;
+    private String color;
+
```

Comment:
let's add a constructor here to add a possibility to use it in classes which inherit this one